src/commands/close.js

```diff
@@ -0,0 +1,27 @@
+const getConfig = require('../config/get-config.js');
+const config = getConfig();
+module.exports = {
+  prefix: 'close',
+  description: 'Closes the channel.',
+  command: async function (message) {
+    try {
+      const target = message.channel;
+      //check for log channel
+      const log = message.guild.channels.cache.find(
+        (channel) => channel.name === config.LOG_MSG_CHANNEL
+      );
+      //check for user permissions
+      if (!message.member.hasPermission('MANAGE_CHANNELS')) {
+        console.log('Missing permissions.');
+      }
+      if (!log) {
+        console.log('Log channel not found.');
+        return;
+      }
+      target.delete();
```

Comment:
This command would close **any** channel right? Isn't that too extreme? Shouldn't we have some level of checks somewhere to prevent us from randomly closing any channel? 

If we open "temp" channels for suspended users, we should be able to recognize that channel is for suspended users via:
- the name of the channel should have "suspended"
- the channel should be under the "suspended" room, from the config.

Reply:
Command now checks if the channel name includes "suspended", and if the channel category name includes the SUSPEND_CATEGORY value from config. 
If either of these fails, it logs it and ends the command.